Directory.Build.props

```diff
@@ -140,20 +140,6 @@
     <InternalInstallerBaseName>$(RuntimeInstallerBaseName)-internal</InternalInstallerBaseName>
   </PropertyGroup>
 
-  <ItemGroup
-    Condition="'$(DisablePubternalApiCheck)' != 'true'
```

Comment:
Please remove `<DisablePubternalApiCheck>true</DisablePubternalApiCheck>` from the 5 places it appears.